### DISCUSSION TITLE
Is Mockoon available for Linux ARM64?

### DISCUSSION THREAD

**Discussion by ntraider** [mockoon/mockoon#903] [General]
I have an Apple M1 Mac and would like to install Mockoon on a Ubuntu 22.04 VM (for ARM).  The Mockoon Downloads page doesn't specify if its Intel/Arm version for Linux.  These downloads don't seem to work on my VM though so assuming they are for Intel.  Does anyone know if Mockoon is available for Linux ARM64 distros?  Thanks.

**Comment by 255kb** [comment]
Indeed, unless specified otherwise, all downloads are for x86 arch by default. 
I will open an issue to track this. But I may need some help to test the binaries as I don't have this kind of architecture.

**Comment by ntraider** [comment]
Hi Guillaume -
Thank you for replying and for all of your work on the Mockoon tool.  
I appreciate you opening an issue to track this.  I'd be happy to test some binaries when those become available. 
My thoughts are with Apple M1 at 40% growth each quarter and many web developers using Mac laptops (including my team) - ARM will increasingly become an important platform to support for tools like Mockoon. Much of my development team often use Linux VMs to do much of their development as it is sandboxed and more configurable.  But with any new platform, the M1 ARM architecture has been a major pain just to figure out if software natively works with it.  Rosetta helps with most software on the OSX side, but not at all yet for virtual machines yet.  
If possible, please consider adding a Intel/ARM info to the Mockoon website for the Linux Download.  There is an Intel/Apple Silicon specification, but not for Linux downloads.  Just letting your users know that only x86 version only for Linux saves people a lot of time, frustration, and probably some emails and forum posts.
Thank you again for your work,
Arnie

    On Tuesday, January 3, 2023 at 06:48:41 AM CST, Guillaume ***@***.***> wrote:  
 
 


Indeed, unless specified otherwise, all downloads are for x86 arch by default.
I will open an issue to track this. But I may need some help to test the binaries as I don't have this kind of architecture.

**Reply by 255kb** [reply]
Sure, I agree. We need to support as much platforms and archs as possible. 
In the meantime I will improve the website download page :)
